ts/src/idex.ts

```diff
@@ -751,6 +752,33 @@ export default class idex extends Exchange {
             const currencyId = this.safeString (entry, 'symbol');
             const code = this.safeCurrencyCode (currencyId);
             const precision = this.parseNumber (this.parsePrecision (this.safeString (entry, 'exchangeDecimals')));
+            const networks = {};
+            const supportedChains = this.safeValue (this.options, 'supportedBlochchains', []);
```

Comment:
@ttodua I'm a bit confused by this logic, what if the user adds a new value to `supportedBlockchains` ?

Reply:
hm, I will revise IDEX again, it was several months ago when I did that. Ok, let's draft this PR, ignore this pr for now, until I'll ping you.